parameter/FloatingPointParameterType.cpp

```diff
@@ -193,8 +193,10 @@ void CFloatingPointParameterType::setOutOfRangeError(
         ostrStream << "real range [" << _fMin << ", " << _fMax << "]";
     } else {
 
-        uint32_t uiMin = reinterpret_cast<const uint32_t&>(_fMin);
-        uint32_t uiMax = reinterpret_cast<const uint32_t&>(_fMax);
+        uint32_t uiMin;
+        utility::binaryCopy(_fMin, uiMin);
+        uint32_t uiMax;
+        utility::binaryCopy(_fMax, uiMax);
```

Comment:
Lets pretend that binary returns the copy:
~~~cpp
auto uiMin = utility::binaryCopy<uint32_t>(_fMin)
auto uiMax = utility::binaryCopy<uint32_t>(_fMax)
~~~
I prefer to initialize variable on declaration.